scripts/environment_json.sh

```diff
@@ -21,13 +21,14 @@ PLATFORM=
 #       src/electron/build_action.sh for more info).
 
 function usage () {
-  echo "$0 [-r] [-h]" 1>&2
+  echo "$0 [-r] [-h] semver" 1>&2
+  echo "  semver: semantic app version"
   echo "  -r: use prod Sentry keys" 1>&2
   echo "  -p: platform (android, ios, osx, browser, windows, linux, or dev)" 1>&2
   echo "  -h: this help message" 1>&2
   echo 1>&2
   echo "Examples:" 1>&2
-  echo "  $0 -p android -r" 1>&2
+  echo "  $0 1.2.3 -p android -r" 1>&2
```

Comment:
I'm iffy on this, but we -could- infer the platform and type from the updated semver tag - how do you feel about this?